docusaurus-docs/src/pages/index.js

```diff
@@ -31,66 +31,129 @@ function HomepageHeader() {
   );
 }
 
-function HomepageVideo() {
-  const {siteConfig} = useDocusaurusContext();
-  return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <img
-          src={useBaseUrl('/img/conda-store-ui.webp')}
-          className={styles.video}
-        />
-        <br></br>
-      </div>
-    </header>
-  );
-}
+const ProjectsList = [
+  {
+    title: '📦 conda-store',
+    link: '/conda-store/introduction',
+    description: (
+      <>
+        Core library that provides key features through a REST API
+      </>
+    ),
+  },
+  {
+    title: '💻 conda-store UI',
+    link: '/conda-store-ui/introduction',
+    description: (
+      <>
+        User-friendly frontend to access conda-store features in a React application
+      </>
+    ),
+  },
+  {
+    title: '🪐 JupyterLab extension',
+    link: '/jupyterlab-conda-store/introduction',
+    description: (
+      <>
+        Provides conda-store-ui frontent within the JupyterLab interface
```

Comment:
```suggestion
    JupyterLab interface that provides conda-store-ui frontend
```
Changed it to have the same structure as other two